tests/jsmarionette/client/sockit-to-me/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "build/Release/sockit",
 
   "devDependencies": {
-    "node-gyp": "~0.10.6"
+    "node-gyp": "3.0.1"
```

Comment:
Sure, I think what I'm suggesting would accomplish that just fine. NPM will hand over the right version based on the engine being used. We just need to put the right engine definitions in (which are the ones I stated). If we're really paranoid about a new node version breaking sockit-to-me then I guess marking it as running with node 0.12.x and ^4.0.0 would do the trick. No need to get crazy with version tracking.